packages/theme-product/theme-product.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Find a match in the project JSON (using a ID number) and return the variant (as an Object)
+ * @param {Object} product Product JSON object
+ * @param {Number} value Accepts Number (e.g. 6908023078973)
+ * @returns {Object} The variant object once a match has been successful. Otherwise null will be return
+ */
+export function getVariantFromId(product, value) {
+  _validateProductStructure(product);
+
+  if (typeof value === 'number') {
+    var result = product.variants.filter(function(variant) {
+      return variant.id === value;
+    });
+
+    return result[0] || null;
+  }
+
+  throw new TypeError(`${value} is not a Number.`);
```

Comment:
This reads a bit weird. Lets throw the exception in an if and keep the success case outside an if.

```
if (typeof value !== 'number') {
  throw new TypeError(`${value} is not a Number.`);
}

var result = product.variants.filter(function(variant) {
  return variant.id === value;
});

return result[0] || null;